visual-inspection/inspection-backend/inspection/explainer/explainers/visuallime_.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import tensorflow as tf
+from xaidemo.tracing import add_span_attributes, traced
+
+from visualime.explain import explain_classification, render_explanation
+
+from .config import LIMEConfiguration
+
+
+def mark_boundaries(img: np.ndarray,
+                    mask: np.ndarray,
+                    color: np.array = np.array([255, 255, 0, 180])
+                    ) -> np.ndarray:
+    # TODO mark only the boundaries adjacent to the highlighted sgements
+    for i in range(1, mask.shape[0] - 1):
+        for j in range(1, mask.shape[1] - 1):
+            if mask[i, j] != mask[i - 1, j]:
+                img[i, j] = img[i, j] * (255 - color[3]) / 255 + color * color[3] / 255
+            elif mask[i, j] != mask[i, j - 1]:
+                img[i, j] = img[i, j] * (255 - color[3]) / 255 + color * color[3] / 255
+    return img
+
+
+@traced(label="compute_explanation", attributes={"explanation.method": "visuallime"})
+def visuallime_explanation(input_img: np.ndarray,
```

Comment:
````suggestion
def visualime_explanation(input_img: np.ndarray,
````
VisuaLIME hat nur ein "L" in der Mitte :-)